MetaMask/metamask-extension
# fix: UX: Remove total account balance from AccountListItem

## **Description**

Removes the total balance which has been identified as a performance problem, reverting to showing native currency balance.

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/25226?quickstart=1)

## **Related issues**

Fixes:

## **Manual testing steps**

1. Open account menu
2. No longer see account total balance or AvatarGroup, instead see native currency balance

## **Screenshots/Recordings**



### **Before**



### **After**

<img width="1011" alt="SCR-20240611-nrym" src="https://github.com/MetaMask/metamask-extension/assets/46655/49ca694c-3eb6-4921-b95f-a7073d586bc7">



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
